Enhance abstract skill for DITA short descriptions (#4814)
theforeman/foreman-documentation: .cursor/skills/abstract/references/concept.md

# Abstract instructions: concept modules

> When the module has `:_mod-docs-content-type: CONCEPT`, follow these rules for the abstract (short description):

Reviewer: ```suggestion
When the module is a concept module (is named `con_*.adoc` or includes `:_mod-docs-content-type: CONCEPT` as its first line), follow these rules for the abstract (short description):
```
This is a more natural description -- "concept module" -- which is better for humans reading the skill.

> - Introduce the concept and provide a concise answer to the question "What is this?" and in some cases "Why do I care about this?"

Reviewer: ```suggestion
- Introduce the concept and provide a concise answer to the question "What is this?" and in some cases "How is this knowledge useful to the user?"
```

There are two parts of my suggestion: replacing "I" with "user" and using more precise wording than the non-technical "care about".

> - If the concept is unfamiliar, start with a brief definition.

Reviewer: ```suggestion
```
This is already defined on line 5 -- "What is this?"

> - Avoid using the short description to lead in or build up to a topic.

Reviewer: What does this mean? A short description does introduce a topic.

Author: It's supposed to summarize the topic, not introduce it.

> - Include the main point of the concept topic.

Reviewer: ```suggestion
```
This duplicates line 5 -- "Why do I care about this?"/"How is this knowledge useful to the user?"
